code/modules/antagonists/eldritch_cult/eldritch_antag.dm

```diff
@@ -210,33 +219,6 @@
 // Objectives //
 ////////////////
 
-/datum/objective/stalk
```

Comment:
do obsesseds use this objective?

Reply:
they use their cuatom versiom of this, arm requested removal of this since it made obsesses less unique, i totally ubderstood what he ment and replaced it with hijack and glorious death